code/game/machinery/doors/airlock.dm

```diff
@@ -123,12 +123,15 @@
 	var/abandoned = FALSE
 	var/cutAiWire = FALSE
 	var/autoname = FALSE
-	var/doorOpen = 'sound/machines/airlock.ogg'
-	var/doorClose = 'sound/machines/airlockclose.ogg'
+	var/doorOpen = 'sound/machines/doors/airlock_open.ogg'
+	var/doorClose = 'sound/machines/doors/airlock_close.ogg'
```

Comment:
I'm not sure if the new opening/closing sounds work as well as the current ones with the door animation timing.
<hr>

The opening/closing animations both last 0.6 seconds, and both of the current sound effects finish after around 0.75 seconds:
*(Each vertical line is 0.25s, 0.6s is marked by the red line)*
![image](https://user-images.githubusercontent.com/57483089/169138125-06675763-ad47-40e9-843f-6fcbcb5c55f7.png)
They're not perfectly timed, but it's close enough that it feels pretty synchronized ingame.

Compared to the Baystation sounds:
*(Each vertical line is 0.25s, 0.6s is marked by the red line)*
![image](https://user-images.githubusercontent.com/57483089/169138522-d17b036c-5f4f-4d24-8723-fe81410fec30.png)
I don't know if they have different door animations on Bay, but both of the new sounds last twice as long as it takes for the door to open or close (finishing at 1.25 seconds avg), and at least to me personally it makes it sound disconnected.